x-pack/plugins/monitoring/public/alerts/missing_monitoring_data_alert/missing_monitoring_data_alert.tsx

```diff
@@ -16,6 +16,9 @@ export function createMissingMonitoringDataAlertType(): AlertTypeModel {
     name: ALERT_DETAILS[ALERT_MISSING_MONITORING_DATA].label,
     description: ALERT_DETAILS[ALERT_MISSING_MONITORING_DATA].description,
     iconClass: 'bell',
+    documentationUrl(docLinks) {
+      return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/kibana/${docLinks.DOC_LINK_VERSION}/kibana-alerts.html`;
```

Comment:
We recently merged in doc changes to add these alerts: https://github.com/elastic/kibana/pull/80922

Can you add `#kibana-alerts-missing-monitoring-data` to the end of the url?

Reply:
Fixed in https://github.com/elastic/kibana/pull/81526/commits/21aac5eb57803e9df9ff83b2ee1f28d132bdcf5e.